packages/instant/src/util/asset_data.ts

```diff
@@ -0,0 +1,18 @@
+import { AssetProxyId } from '@0xproject/types';
+
+import { assetMetaData } from '../data/asset_meta_data';
+
+// TODO: tests for this
+export const bestNameForAsset = (assetData: string | undefined, defaultString: string) => {
```

Comment:
I like the way this is structured but I'd like to follow our somewhat soft convention of exporting one thing from file.
So the api would be `assetDataUtils.bestName()`

Reply:
I wonder if this convention could be bad for optimizing builds.  Not sure.  Cool w/ keeping convention for now though